lms/services/jstor.py

```diff
@@ -94,13 +104,11 @@ def metadata(self, article_id: str):
 
         :param article_id: A JSTOR article ID or DOI
         """
-        doi = self._doi_from_article_id(article_id)
-        response = self._api_request(_format_uri("/metadata/{}", doi))
```

Comment:
This pattern of:

* `_doi_from_article_id`
* `_format_uri` (passed only `doi`)

Was repeated before every call to `_api_request`, and each of these functions were only used in this way.

Generally this is a good sign that you should move things inside the target function.